src/components/components.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
- [ ] **According to the project requirements you should create Inside components, a new Calculator.js file, and should be a **functional** component**. Please that as a developer it is very important to always follow the requirements because they establish a foundation for product vision, scope, cost, and schedule and they ultimately must target finished product quality and performance. Please fix it :+1: